### ISSUE TITLE
example won't working on prisma 1.7

### ISSUE THREAD

**Issue by wuzhuzhu** [maxdarque/prisma-docker#6]
Trying to follow your instruction to connect my Prisma server to an external database, No luck.
prisma upgraded to 1.7 changes docker-compose.yml, which move environments to | taged scala arguments.
Could you please upgrade the example to latest prisma version? Thanks a lot

**Comment by maxdarque** [comment]
Yes there were quite a few config changes with the release of prisma 1.7 and I need to update the repo to reflect these changes. I'll have a look at this at some point over the next two weeks.

**Comment by szhang-ezypay** [comment]
The latest prisma is 1.17. Is there any chance you can update your example? Thanks a lot.

**Comment by maxdarque** [comment]
Sorry guys. I've switched jobs and will be focusing something new. This is going to get parked and will be unmaintained.
